index.html

```diff
@@ -0,0 +1,284 @@
+          <li class="nav-social-item mr-1"><i class="fab fa-facebook-f"></i></li>
+          <li class="nav-social-item mr-1"><i class="fab fa-twitter"></i></li>
+          <li class="nav-social-item mr-1">English</li>
+          <li class="nav-social-item mr-1">My Page</li>
+          <li class="nav-social-item">Logout</li>
+        </ul>
+      </div>
+
+      <nav id="navbar" class="nav-bar">
+        <div id="bars-menu" class="bars-menu d-none-desk">
+          <i class="fas fa-bars fa-2x"></i>
+        </div>
+        <div class="nav-list-container bc-white d-flex d-justify-between d-align-center d-none desk-w">
+          <a href="./index.html"><img class="nav-logo" src="./img/logos/ant_logo.png" alt=""></a>
+          <ul class="nav-list d-justify-between d-align-center d-none">
+            <li><a class="nav-links default-family fc-orange fw-700" href="./about.html">About</a></li>
+            <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">Program</a></li>
+            <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">Join</a></li>
+            <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">Sponsor</a></li>
+            <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">News</a></li>
+            <li><a class="nav-links nav-list-item-btn default-family fc-orange fw-700" href="#">MicroAnt Campain</a></li>
+          </ul>
+        </div>
+      </nav>
+    </section>
+
+    <section id="headline-section" class="headline-section d-flex d-justify-center pl-1">
+      <div class="headline-container d-flex d-column">
+        <h1 class="headline-desk-size headline-title default-family fw-900 fc-orange fs-48 mb-2">
+          MICROANT FILM FESTIVAL
+        </h1>
+        <p class="headline-desk-size headline-pbox default-family fw-500 fs-14 fc-greyish mb-1">
+          Enthusiastic MicroAnt's! The day has come, enjoy a lot of great movies
+          remotely in the best film festival that you can participate off choose
+          your day and watch with all of yours microants friends!
+        </p>
+        <h2
+          class="
+            headline-desk-size
+            headline-support
+            default-family
+            fw-900
+            fc-dark-greyish
+            fs-24
+            mb-1
+          "
+        >
+          2021.8.8(SAT) ~ 9(SUN)
+        </h2>
+        <h2
+          class="headline-desk-size headline-support default-family fw-500 fc-greyish fs-14 mb-1"
+        >
+          @ Zoom Conference Meeting, EveryWhere in the Globe.
+        </h2>
+      </div>
+    </section>
+
+    <main id="main-section">
+      <section id="main-program" class="main-program">
+
+        <div id="programs-out-container" class="programs-out-container d-flex d-column d-align-center">
+
+          <div
+            id="programs-header-container"
+            class="d-flex d-column d-align-center mb-1"
+          >
+
```

Comment:
- [ ] Kindly delete this blank line 👍 Please keep in mind that we're only adding blank lines for a reason such as separating two sections or two large pieces of code apart from each other for readability. 🙏 

_You can read about this tip and more on this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Do%20not%20add%20blank%20lines%2C%20spaces%2C%20or%20indentations%20without%20a%20reason.)._
_Please apply this request to all similar situations through the pages._